src/settings/target/mod.rs

```diff
@@ -120,6 +129,106 @@ impl Manifest {
         Ok(manifest)
     }
 
+    pub fn generate(
+        name: String,
+        target_type: TargetType,
+        config_path: PathBuf,
+        site: bool,
+    ) -> Result<Manifest, failure::Error> {
+        let mut manifest = Manifest::default();
```

Comment:
Why did we get rid of the nice struct init logic from before?

Reply:
idk it sounded like a good idea to use default but i'm not using it well here.

Reply:
this can live for now in the interest of getting RC out!!